primary/src/proposer.rs

```diff
@@ -135,7 +135,23 @@ impl Proposer {
         self.last_parents = Certificate::genesis(&self.committee);
     }
 
-    // Main loop listening to incoming messages.
+    /// Compute the timeout value of the proposer.
+    fn timeout_value(&self) -> Instant {
```

Comment:
Can we somehow test the changes of this? Also what would be the way to see whether this actually makes an improvement? Can you think of some metric we could introduce that would show this?